modules/nf-core/cat/fastq/main.nf

```diff
@@ -14,6 +14,9 @@ process CAT_FASTQ {
     tuple val(meta), path("*.merged.fastq.gz"), emit: reads
     path "versions.yml"                       , emit: versions
 
+    publish:
+    reads   >> 'cat/fastq'
```

Comment:
How would a pipeline developer turn this off? If I include this module from nf-core/modules in my own pipeline, but don't want to publish the output from this, what can I do?